packages/bits/src/lib/popup/popup-container.component.ts

```diff
@@ -26,7 +26,7 @@ import { PositionService } from "../../services/position.service";
     encapsulation: ViewEncapsulation.None,
 })
 
-/* tslint:disable:use-host-property-decorator */
+/* eslint-disable */
```

Comment:
is there a rule name that can be specified here?

Reply:
good catch, the **@angular-eslint/no-host-metadata-property** rule it is. Added